lisa/base_tools/wget.py

```diff
@@ -49,13 +49,13 @@ def get(
         # So that here can use the corresponding path format.
         if file_path:
             # create folder when it doesn't exist
-            self.node.shell.mkdir(PurePosixPath(file_path), exist_ok=True)
+            self.node.shell.mkdir(Path(file_path), exist_ok=True)
```

Comment:
Why makes this change? It maybe a remote path, and supports only posix, so it's PurePosixPath.

Reply:
This wont be the remote path. We are doing wget on node to get the source for installation of goLang having specific version.

I changed this as assert failure was happening at shell.py on line 666 as it expect Path instance and was getting PurePosixPath.

Path and PurePosixPath are child classes of PurePath.

Reply:
I will try to run again without this change ny reverting it and will make sure.